dotcom-rendering/scripts/jsonSchema/schema.mjs

```diff
@@ -16,6 +16,7 @@ const program = TJS.getProgramFromFiles(
 		path.resolve(`${root}/src/types/newslettersPage.ts`),
 		path.resolve(`${root}/src/types/editionsCrossword.ts`),
 		path.resolve(`${root}/src/feFootballDataPage.ts`),
+		path.resolve(`${root}/src/feCricketMatch.ts`),
```

Comment:
Can we put this in the `frontend` directory and name it after the core type it exports?

```suggestion
		path.resolve(`${root}/src/frontend/feCricketMatchPage.ts`),
```

See #13372 #13622 and #13703 for why the `frontend` directory and the reasons for the inconsistency at the moment.

Reply:
Should we do the same for the football one(s)?